### ISSUE TITLE
License 

### ISSUE THREAD

**Issue by BThoom** [bedimcode/responsive-portfolio-website-Alexa#34]
Is this open source? Can I use this code to create my portfolio?

**Comment by Fernanda-Dantas** [comment]
I don't think so. I used the code do create my portfolio (I did a fork) but now I'm trying to create a repository here on Github but my access has been denied. The message that appears is that beimcode denied access to me. It's too bad because my portfolio turned down so pretty and I just wasted 6 months trying to do it :(
